### PR TITLE
Add boilerplate verify scripts

### PR DESCRIPTION
**Pull request by duglin**
This is an alterntaive to #195.

Rather than spending hours discussing vendoring vs subtrees, I decided it
might be easier to allow people to see the differences and compare them
since it only took a few minutes to throw this together.

I don't think this is fundamentally different from #195 except it vendors
in the other repo instead of using a git subtree.

This one should pass CI though.

I wanted to delete even more files but I wasn't sure which ones were still needed so I tried to just limit the deletes to the same ones in #195.  We can clean up the rest later, or in this PR if someone knows for sure which are ok to remove.

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by pmorie** [comment]
I think I've kind of given up on the idea of repo-infra for now.  I don't personally have the bandwidth at this point to make something durable that will meet everyone's needs.  So, we can just copy the code for now and refactor later, IMO.

**Comment by pmorie** [comment]
Before this goes in, I would like:

1.  `repo-infra` to be updated with documentation for using it via vendor
2.  The glide cache to be ignored without having to be called in the pipeline

**Comment by duglin** [comment]
done: https://github.com/kubernetes/repo-infra/pull/3

**Comment by krancour** [comment]
After https://github.com/kubernetes/repo-infra/pull/3 _and_ `glide.yaml` in this PR is updated appropriately to reference that latest version of repo-infra, then this LGTM.

**Comment by duglin** [comment]
ok - the repo-infra PR is merged and this one has been rebased/updated.

File: Makefile [hunk 1]
```diff
@@ -118,6 +118,10 @@ verify: .init .generate_files
 	  ($(DOCKER) golint --set_exit_status $$i && \
 	   $(DOCKER) go vet $$i )|| exit $$? ; \
 	done || false
+	@echo Running repo-infra verify scripts
+	$(DOCKER) vendor/github.com/kubernetes/repo-infra/verify/verify-boilerplate.sh --rootdir=. | grep -v zz_generated > .out 2>&1 || true
```
Comment (pmorie) [inline Makefile]: The generated files should actually be added with a correct boilerplate, but I understand this for now.

**Comment by pmorie** [comment]
LGTM

**Comment by krancour** [comment]
LGTM

File: Makefile (hunk 1 above)
Comment (duglin) [inline Makefile]: yup - agreed and a follow-on PR will fix those

**Comment by bmelville** [comment]
LGTM pending good CI.

**Comment by duglin** [comment]
just fixed the CI - should be good in a few minutes

**Pull request merged**